.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Ask a question
```

Comment:
if we will be adding questions variant i would recommend in text to use ether discussions https://github.com/oneapi-src/oneDAL/discussions or UXL oneDAL Slack channel

Reply:
good call, I think dropping question.md is a good idea.